### DISCUSSION TITLE
Itinéraires routiers

### DISCUSSION THREAD

**Discussion by Allmania** [osmandapp/OsmAnd#19370] [Q&A]
Bonjour à l'équipe Osmand,
Je suis un **contributeur actif à OpenStreetMap Gabon** et j'utilise **Osmand**+ au quotidien. J'ai créé un itinéraire routier (RN9) avec les relations et combinaisons associées à la route comme indiqué par Wiki OpenStreetMap. Lorsque j'arrive à Osmand+ et que je clique sur un bouclier pour consulter les informations sur le tracé de la RN9, rien n'apparaît. Pourriez-vous m'aider s'il vous plaît.
les informations qui doivent affichées sont selon un article de la page d'aide du [site d'OsmAnd](https://osmand.net/docs/user/map/tracks-on-map) 
**Images**:
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/78262248/6ab0f89f-869a-4211-8039-e1ff10872f60"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/78262248/92ecc78b-2a53-4e73-9fa1-5937c6268e93"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/78262248/ced575f5-d12c-4969-a5fe-cc2ba1e48502"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by DmitryAlexei** [comment]
Hi, and thank you very much for contributing to OSM!
Please note that the information in the article you mentioned is about routes (see link [wiki OSM]( https://wiki.openstreetmap.org/wiki/Relation:route)), not ways.

**Reply by Allmania** [reply]
Salut à vous et merci de m'avoir répondu. Dans ma question, je demande pourquoi **les informations associées à l'itinéraire RN9 n'apparait pas** dans OsmAnd lorsque **l'on clique sur le bouclier**? Voici ma préoccupation.

**Reply by DmitryAlexei** [reply]
This information will be available while driving, not by tap. Or can you obtain this information in other places of the app? If so, please provide screenshots.
